spec/services/export_files/packager_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+module ExportFiles
+  RSpec.describe Packager do
+
+    describe '#build!' do
+
+      subject { Packager.new(package, top_work.id, ability: ability) }
```

Comment:
Packager's parms for initialize have (package, work, ability:) but work.id is passed in.